From 47f94c38ec9534b0364ecb8beba67a7602745dbc Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@apache.org>
Date: Tue, 24 Jul 2012 22:08:27 +0000
Subject: [PATCH 0249/1357] MAPREDUCE-4465. Update description of yarn.nodemanager.address property. (bowang via tucu)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/trunk@1365322 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit b3a7f95190351d0921f037fcf86d97d9b9034ed4)

Conflicts:

	hadoop-mapreduce-project/CHANGES.txt
---
 .../src/main/resources/yarn-default.xml            |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hadoop-mapreduce-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml b/hadoop-mapreduce-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml
index 6e8df10..bed12e1 100644
--- a/hadoop-mapreduce-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml
+++ b/hadoop-mapreduce-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml
@@ -241,7 +241,7 @@
 
   <!-- Node Manager Configs -->
   <property>
-    <description>address of node manager IPC.</description>
+    <description>The address of the container manager in the NM.</description>
     <name>yarn.nodemanager.address</name>
     <value>0.0.0.0:0</value>
   </property>
-- 
1.7.0.4

